### PR TITLE
Fix incomplete Display after swipe up the screen to main

### PR DESCRIPTION
**Pull request by RakeshReddyGunda**
menu in portrait Mode.

Jira: 59439.
Test: Build passes on Android.

Signed-off-by: reddyr3x <rakeshx.reddy@intel.com>

### CODE REVIEW

File: common/core/overlaylayer.cpp [hunk 1]
```diff
@@ -263,6 +255,15 @@ void OverlayLayer::InitializeState(HwcLayer* layer,
     }
   }
 
+ // In case of layer using blening we need to force partial clear. Otherwise
```
Comment (kalyankondapally) [inline common/core/overlaylayer.cpp]: Why does moving the code chunk here fix the issues ?

Comment (stripes416) [inline common/core/overlaylayer.cpp]: Also, while you didn't write the comment, just moved it, the misspelling should be fixed.
/blening/blending/

Comment (RakeshReddyGunda) [reply common/core/overlaylayer.cpp]: swap should be  done on layer-> GetLayerDamage as we are rotating display to 90.
But in this case the swap was done on layer->GetDisplayFrame  which caused incomplete screen.

Comment (kalyankondapally) [reply common/core/overlaylayer.cpp]: hmm but u still use DisplayFrame right ?

Comment (RakeshReddyGunda) [reply common/core/overlaylayer.cpp]: Yes we use DisplayFrame after swapping the surface damage co-ordinates.

Comment (md-sami) [reply common/core/overlaylayer.cpp]: After the rotation is done, the source layer dimensions are already updated, so why do we need to do swap again?

Comment (md-sami) [reply common/core/overlaylayer.cpp]: After the rotation is done, the source layer dimensions are already updated, so why do we need to do swap again?

**Pull request closed without merging**